internal/cli/server/api_service.go

```diff
@@ -91,3 +101,27 @@ func (s *Server) BorBlockReceipt(ctx context.Context, req *protobor.ReceiptReque
 
 	return &protobor.ReceiptResponse{Receipt: ConvertReceiptToProtoReceipt(receipt)}, nil
 }
+
+func getRpcBlockNumberFromString(blockNumber string) (rpc.BlockNumber, error) {
+	switch blockNumber {
+	case "latest":
+		return rpc.LatestBlockNumber, nil
+	case "earliest":
+		return rpc.EarliestBlockNumber, nil
+	case "pending":
+		return rpc.PendingBlockNumber, nil
+	case "finalized":
+		return rpc.FinalizedBlockNumber, nil
+	case "safe":
+		return rpc.SafeBlockNumber, nil
+	default:
+		blckNum, err := strconv.ParseUint(blockNumber, 10, 64)
```

Comment:
Should we use hex for block number as other RPC endpoints do [here](https://github.com/maticnetwork/bor/blob/16268f6608d09d5b6d0cd0e43f5ec076c3b92616/rpc/types.go#L110)?

Reply:
Addressed [here for bor](https://github.com/maticnetwork/bor/pull/1374/commits/10d9594b52ee02dcdebc1a0284993b6a89046c72) and [here for heimdall](https://github.com/maticnetwork/heimdall/pull/1204/commits/1f535e5c8b71793fafaad816815fa02fcb2e9286).
Just tested, everything works!
Thank you